### PR TITLE
fix(workflows): fix kyb workflow, duplicate business creation on run

### PR DESCRIPTION
**Pull request by alonp99**

### CODE REVIEW

File: packages/workflow-core/src/lib/workflow-runner.ts [hunk 1]
```diff
@@ -420,14 +420,21 @@ export class WorkflowRunner {
...
 
           if (state.done) {
             console.log('Reached final state');
           }
 
+          if (state.tags.has('failure')) {
+            console.log('Reached failure state', {
+              correlationId: context?.entity?.id,
+              ballerineEntityId: context?.entity?.ballerineEntityId,
+            });
```
Comment (liorzam) [inline packages/workflow-core/src/lib/workflow-runner.ts:435]: use logger instead console.log

File: packages/workflow-core/src/lib/plugins/external-plugin/api-plugin.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export class ApiPlugin {
     this.method = pluginParams.method;
     this.headers = {
       'Content-Type': 'application/json',
+      accept: 'application/json',
```
Comment (liorzam) [inline packages/workflow-core/src/lib/plugins/external-plugin/api-plugin.ts:26]: can remove

File: packages/workflow-core/src/lib/workflow-runner.ts [hunk 3]
```diff
@@ -420,14 +420,21 @@ export class WorkflowRunner {
...
         if (state.changed) {
           console.log('Transitioned into', state.value);
 
           if (state.done) {
             console.log('Reached final state');
           }
 
+          if (state.tags.has('failure')) {
+            console.log('Reached failure state', {
+              correlationId: context?.entity?.id,
```
Comment (liorzam) [inline packages/workflow-core/src/lib/workflow-runner.ts:433]: entity id has to be present why do you access with question mark?

**Review by liorzam — APPROVED** [review]

File: packages/workflow-core/src/lib/workflow-runner.ts (hunk 1 above)
Comment (alonp99) [reply packages/workflow-core/src/lib/workflow-runner.ts:435]: no logger yet in the wf-core

File: packages/workflow-core/src/lib/plugins/external-plugin/api-plugin.ts (hunk 2 above)
Comment (alonp99) [reply packages/workflow-core/src/lib/plugins/external-plugin/api-plugin.ts:26]: It might be beneficial for other API, no downside here

**Pull request merged**